lib/cluster/index.js

```diff
@@ -209,6 +209,29 @@ Cluster.prototype.disconnect = function (reconnect) {
 };
 
 /**
+ * Quit the cluster gracefully.
+ *
+ * @param {function} callback
+ * @return {Promise} the count of nodes that have been disconnected
+ * @public
+ */
+Cluster.prototype.quit = function (callback) {
+  this.setStatus('disconnecting');
+
+  this.manuallyClosing = true;
+
+  if (this.reconnectTimeout) {
+    clearTimeout(this.reconnectTimeout);
+    this.reconnectTimeout = null;
+  }
+  return Promise.all(this.nodes().map(function (node) {
+    return node.quit();
+  })).then(function (res) {
+    return res.length;
```

Comment:
Should we return the number of the node been disconnected or just `"OK"` to keep in line with the result of native quit.